PowerType/Dictionaries/npm.ps1

```diff
@@ -37,7 +64,156 @@
+                    Description = "Installs the SQLite binding for Node.js";
+                },
+                [ValueParameter]@{
+                    Keys = @("better-sqlite3");
+                    Name = "better-sqlite3";
+                    Description = "Installs a faster and simpler version of SQLite";
+                },
+                [ValueParameter]@{
+                    Keys = @("redis");
+                    Name = "redis";
+                    Description = "Installs the modern Redis client for Node.js";
+                },
+                [ValueParameter]@{
+                    Keys = @("ioredis");
+                    Name = "ioredis";
+                    Description = "Installs a robust Redis client with more features";
+                },
+                [ValueParameter]@{
+                    Keys = @("tedious");
+                    Name = "tedious";
+                    Description = "Installs the official Microsoft SQL Server (MSSQL) driver for Node.js";
+                },
+                [ValueParameter]@{
+                    Keys = @("sequelize");
+                    Name = "sequelize";
+                    Description = "Installs Sequelize, a promise-based ORM supporting PostgreSQL, MySQL, MariaDB, SQLite and SQL Server";
+                },
+                [ValueParameter]@{
+                    Keys = @("typeorm");
+                    Name = "typeorm";
+                    Description = "Installs TypeORM, a feature-rich ORM supporting Active Record and Data Mapper patterns for TypeScript and JavaScript";
+                },
+                [ValueParameter]@{
+                    Keys = @("prisma");
+                    Name = "prisma";
+                    Description = "Installs Prisma, a modern ORM with auto-generated queries, migrations and type-safe client for Node.js";
+                },
+                [ValueParameter]@{
+                    Keys = @("objection");
+                    Name = "objection";
+                    Description = "Installs Objection.js, an SQL-friendly ORM built on top of Knex.js for flexible query building";
+                },
+                [ValueParameter]@{
+                    Keys = @("bookshelf");
+                    Name = "bookshelf";
+                    Description = "Installs Bookshelf.js, an ORM built on top of Knex.js that provides relations and plugins";
+                },
+                # finish common database drivers and ORMs 
+                [ValueParameter]@{
+                    Keys = @("typescript");
+                    Name = "typescript";
+                    Description = "Installs TypeScript locally";
+                },
+                [ValueParameter]@{
+                    Keys = @("-D", "--save-dev");
+                    Name = "-D";
+                    Description = "Installs the package as a development dependency (saved under devDependencies)";
+                },
+                [ValueParameter]@{
```

Comment:
Shouldn't this be a FlagParameter?

Reply:
Have you tried to move the package list into a `ValueParameter` without a key and a `StaticSource` with `SourceItems` like we do in the Git repo?
https://github.com/AnderssonPeter/PowerType/blob/2e024e6ea64d5cb047ad4d84474e65fe6d383e4a/PowerType/Dictionaries/git.ps1#L570